### PR TITLE
Fix index typo

### PR DESCRIPTION
**Pull request by Xeonacid**
## Done

[List of work items including drive-bys]
- replace `一下` with `以下` 
## QA

[List of steps to QA the new features or prove the bug has been resolved]
- Just view the site 😄 
## Issue / Card

[List of links to Github issues/bugs and cards if needed - e.g. `Fixes #1`]

## Screenshots

[if relevant, include a screenshot]

### CODE REVIEW

**Comment by SirSamTumless** [comment]
@zprood Could you give this a review?

**Comment by zprood** [comment]
@SirSamTumless The PR looks good, I haven't noticed these words are wrong. :D

File: templates/about/index.html [hunk 1]
```diff
@@ -81,7 +81,7 @@ <h3>认证的 Ubuntu 分支版本</h3>
         <li class="p-list__item is-ticked"><a href="https://ubuntu-mate.org/">Ubuntu MATE</a> — 具有 MATE 桌面环境的 Ubuntu</li>
       </ul>
       <h3>本地化</h3>
-      <p>出于空间方面的考虑，标准 Ubuntu 映像仅随附了少数几种语言。用户可以通过系统设置或使用网络安装器来添加其他语言的支持。标准映像直接提供对一下相关语言的完整支持。</p>
+      <p>出于空间方面的考虑，标准 Ubuntu 映像仅随附了少数几种语言。用户可以通过系统设置或使用网络安装器来添加其他语言的支持。标准映像直接提供对以下相关语言的完整支持。</p>
```
Comment (Xeonacid) [inline templates/about/index.html:84]:
```suggestion
      <p>出于空间方面的考虑，标准 Ubuntu 映像仅随附了少数几种语言。用户可以通过系统设置或使用网络安装器来添加其他语言的支持。标准映像直接提供对以下相关语言的完整支持：</p>
```
@zprood How about this?

Comment (zprood) [reply templates/about/index.html:84]: I think we need to create a new list for this. Thank you for reporting this.

**Pull request closed without merging**